### ISSUE TITLE
More Info about the TEST Set

### ISSUE THREAD

**Issue by xingener** [epfl-dlab/homepage2vec#1]
Could you share more information about the testing dataset?

Info I know by now:
There are 2 subsets of the testing set.
#1 is with a "balanced setup". There are 14 sub-subsets each of which contains balanced number of positives and negatives for a single label. 
#2 is with a "unbalanced setup". I think all samples are randomly sampled under the original distribution of the dataset. 

My questions:
Could you share all the testing sets you used in the experiments? If so, I could benchmark in exactly the same way.
If not, could you please share more info about:
Number for each sub-subset in #1 subset? Are all sub-subsets share a same negative set?
Is the #2 subset are sampled from the 1.05M dataset which is released in [https://figshare.com/articles/dataset/Language-agnostic_Website_Embedding_and_Classification_Curlie_dataset_/16621669?file=31222348]

Thanks a lot!

**Comment by sylvainlugeon** [comment]
We've evaluated our model as follows.

The 1.05M dataset is composed of unique pairs of _(url, label)_ but can contain multiple occurrence of the same url under different labels. We grouped this dataset by url yielding an input matrix X with the url features and target matrix Y that contains the (possibly many) labels under which the url appear in the dataset.

A 90/10 split is performed on the matrices to obtain the training and testing set. Each class-wise classifier is then evaluated independently. In the _unbalance_ setup, we evaluate each classifier on the whole testing set. In the _balance_ setup, we do the evaluation on a subset of the testing set (one for each classifier), that is composed of:
- all positive samples for that class in the testing set
- the same number of randomly selected negative samples, still from the testing set

Does it answer your question?

Right now, I don't have the cardinalities of the subsets for the _balance_ evaluation nor their indices in the dataset. I'll come back to you in the next weeks if I find some time to regenerate the testing sets.
